client/submodules/live/GlassLiveConnection.cs

```diff
@@ -99,7 +99,6 @@
 
 	%this.send(jettisonStringify("object", %obj) @ "\r\n");
 	%obj.delete();
-	%digest.delete();
```

Comment:
This should actually be moved up into the previous conditional branch, where the digest is created.